app/_kong_plugins/rate-limiting/index.md

```diff
@@ -126,3 +68,41 @@ When a limit is reached, the plugin returns an `HTTP/1.1 429` status code, with
 > The headers `RateLimit-Limit`, `RateLimit-Remaining`, and `RateLimit-Reset` are based on the Internet-Draft [RateLimit Header Fields for HTTP](https://datatracker.ietf.org/doc/draft-ietf-httpapi-ratelimit-headers/) and may change in the future to respect specification updates.
 
 
+
+## Use Casese
+Two common use cases are:
+
+| You need... | Use the following plugin policy strategies... |
+| --------- | ---- | 
+| A high level of accuracy in ?. An example is a transaction with financial consequences. | `cluster` or `redis` | 
+| Protect backend services from overloading that's caused either by specific users or by attacks. ___ accuracy is not as relevant. | `local` |
+
+If the plugin can't retrieve the selected policy, it falls back to [limiting usage by identifying the IP address](#limit-by-ip-address).
+
+### High accuracy use case recommendations
+
+In this scenario, because accuracy is important, the `local` policy is not an option. Consider the support effort you might need
+for Redis, and then choose either `cluster` or `redis`.
+
+You could start with the `cluster` policy, and move to `redis`
+if performance reduces drastically.
+
+Do remember that you cannot port the existing usage metrics from the data store to Redis.
+This might not be a problem with short-lived metrics (for example, seconds or minutes)
+but if you use metrics with a longer time frame (for example, months), plan
+your switch carefully.
+
+### Backend protection use case recommendations
+
+If accuracy is of lesser importance, choose the `local` policy. You might need to experiment a little
+before you get a setting that works for your scenario. As the cluster scales to more nodes, more user requests are handled.
+When the cluster scales down, the probability of false negatives increases. So, adjust your limits when scaling.
+
+For example, if a user can make 100 requests every second, and you have an
+equally balanced 5-node Kong cluster, setting the `local` limit to something like 30 requests every second
+should work. If you see too many false negatives, increase the limit.
+
+To minimize inaccuracies, consider using a consistent-hashing load balancer in front of
+Kong. The load balancer ensures that a user is always directed to the same Kong node, thus reducing
+inaccuracies and preventing scaling problems.
+
```

Comment:
All of this could probably move to the Strategies page that I added here: https://github.com/Kong/developer.konghq.com/pull/49/files#diff-c3a155cc86e9ccee3bcb0ca7d1096a01a71a42ecc73ded8d5d1fb83f861cf748

It's just not merged yet. But you're welcome to add to my PR if you want.

Reply:
done